test/routing/validation.test.js

```diff
@@ -0,0 +1,11 @@
+var test  = require('tape')
+  , rhumb = require('../../src/rhumb')
+
+test('Routing should routes containing //', function(t) {
```

Comment:
Woops.  This was a placeholder test to remind me to test that // throws an error when used as a route.